src/views/33Together/33Together.jsx

```diff
@@ -104,6 +104,7 @@ const PoolTogether = () => {
     let userOdds = calculateOdds(poolBalance, totalDeposits, winners);
     setYourOdds(userOdds);
   }, [poolData, poolBalance]);
+  // TSX-REFACTOR-NOTE: Shouldn't winners be in this array?
```

Comment:
Ah. You made me double check. Winners is part of poolData. That's why I didn't add it separately. What do you think?

Reply:
It would probably be even better if we did `poolData.winners` in the array along with any other variables that are part of the poolData object. This would lead to slightly fewer renders, but if poolData is comprised of totalDeposits and winners, I think it makes sense.